activesupport/test/cache/stores/mem_cache_store_test.rb

```diff
@@ -34,6 +46,56 @@ def setup
   include EncodedKeyCacheBehavior
   include AutoloadingCacheBehavior
 
+  def test_connection_pool
```

Comment:
Do we need to test the behavior of the connection_pool or just asserting that an instance of `ConnectionPool` is being used by the cache store is sufficient?

I'm good with the current tests, but those sleeps worries me. Do you know how much time those two new tests are taking?

Reply:
3 seconds at most per test case using `emulating_latency`.

Reply:
Ah, so it is fine.